doc/guides/writing-tests.md

```diff
@@ -209,6 +209,35 @@ const assert = require('assert');
 const freelist = require('internal/freelist');
 ```
 
+### Assertions
+
+When writing assertions, prefer the strict versions:
+
+* `assert.strictEqual()` over `assert.equal()`
+* `assert.deepStrictEqual()` over `assert.deepEqual()`
+
+When using `assert.throws()`, if possible, provide the full error message:
```

Comment:
If you want to drop "if possible" and just say "provide the full error message", that works for me too. I have no strong opinion either way.  Adding either one would be an improvement to the doc, so 👍 from me whichever way you go on it.